config/locales/en/en.yml

```diff
@@ -161,6 +161,10 @@ en:
           attributes:
             primary_agent:
               same_as_secondary: 'The point person for this project cannot be the same as the second point person'
+        basic_project:
+            attributes:
+              primary_agent:
+                same_as_secondary: 'The primary agent for this project cannot be the same as the secondary agent'
```

Comment:
Can we use the same wording as the above `same_as_secondary`.

Reply:
I do not understand

Reply:
See line 163. Use the same message.

Reply:
I see